tests/driver/use_pch_msvc.c

```diff
@@ -9,14 +9,12 @@
 
 // Check that IWYU fails for MSVC spelling of PCH args.
 
-// IWYU_ARGS: --driver-mode=cl /I . /Yu tests/driver/use_pch_msvc.h
-
-// Clang for some reason creates a precompiler job when /Yu (use PCH) is
-// present. We filter it out, but warn, so expect that warning.
-// IWYU~: ignoring unsupported job type: precompiler
+// IWYU_ARGS: --driver-mode=cl /I . /Yutests/driver/use_pch_msvc.h
```

Comment:
Putting a space there doesn't seem to work. With a space, the test fails with this error:
```
#pragma hdrstop not seen while attempting to use precompiled header
```

Without the space, we get the intended:
```
include-what-you-use does not support PCH
```

This seems consistent with Clang's command line documentation:
```
/Yu<filename>           Load a pch file and use it instead of all code up to and including <filename>
```
https://clang.llvm.org/docs/UsersManual.html

Unless I'm missing something?

Reply:
Huh. OK, then that makes sense. It just looked to me like an incidental change. Thanks for explaining!